treeherder/etl/buildapi.py

```diff
@@ -174,6 +174,16 @@ def transform(self, data, filter_to_project=None, filter_to_revision=None,
             else:
                 log_reference = []
```

Comment:
Now we set/append log_reference in two places, I think we should put the ``log_reference = []`` first, followed by two |if 'foo' in prop: ...| which both log_reference.append().